index.html

```diff
@@ -12,7 +12,6 @@
   </head>
   <body>
     <section class="container bg-light p-0 m-0">
```

Comment:
- [ ] When I open the mobile menu and switch to the desktop screen the mobile menu is still on the screen it should disappear. Please make sure when we switch the screen from mobile to desktop the mobile menu should disappear.
 - You can use media queries for that if the screen cross the min-width you can set your mobile menu `display: none`

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/158642384-1fe15e90-0f29-4d04-8786-06eb79d0290d.png)